numba/tests/test_parfors.py

```diff
@@ -1018,6 +1018,24 @@ def test_impl(n):
         self.assertEqual(parfor_output, py_output)
         self.assertEqual(countParfors(test_impl, (types.int64, )), 0)
 
+    def test_stararg(self):
```

Comment:
RE testing, was there the intention that this should work:
```python
from numba import njit
import numpy as np

@njit(parallel=True)
def foo(*args):
    x = np.random.randn(2, *args)
    return x

print(foo(1, 2))
```
at present this won't lower, it produces:
```
numba/core/base.py", line 616, in get_getattr
    raise NotImplementedError("No definition for lowering %s.%s" % (typ, attr))
NotImplementedError: No definition for lowering UniTuple(int64 x 3).shape
```